performance_tests/consolidate_results.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Script for consolidating results for execution times"""
+
+import csv
+import sys
+
+
+def calculate_statistics(_csv_file):
+    """Calculate min, max and average statistics for execution times in a CSV file."""
+    with open(_csv_file, 'r', encoding='utf-8') as file:
+        reader = csv.reader(file)
+        data = [float(row[0]) for row in reader]
+
+    # Calculate statistics
+    if data:
+        _total_entries = len(data)
+        _average = sum(data) / _total_entries
+        _minimum = min(data)
+        _maximum = max(data)
+        return _total_entries, _average, _minimum, _maximum
+
+    return None
+
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
```

Comment:
This is fine for this project.
Just a heads up that a better maintainable approach would be to use argparse: https://docs.python.org/3/library/argparse.html
I don't think you should go back and change this unless you need to revisit the CLI arguments for some other reason

Reply:
thanks, updated